src/discovery/Akka.Discovery.AwsApi/Ec2/Ec2ConfigurationProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Amazon.EC2;
+using Amazon.Runtime;
+
+namespace Akka.Discovery.AwsApi.Ec2
+{
+    public abstract class Ec2ConfigurationProvider
```

Comment:
Derived Ec2ConfigurationProvider class must have either an empty constructor or a constructor that takes an ExtendedActorSystem as a parameter